oio/xcute/common/backend.py

```diff
@@ -545,6 +559,14 @@ def list_jobs(self, marker=None, limit=1000):
                     # The job can be deleted between two requests
                     continue
 
+                if job_status and job_info['job.status'] != job_status:
+                    continue
+                if job_type and job_info['job.type'] != job_type:
+                    continue
+                if job_lock and not fnmatchcase(
```

Comment:
is it really necessary to use `fnmatchcase` ?

Reply:
With `fnmatchcase`, we allow listing a single service type (`rawx/*`), a single server (`*/127.0.0.1`), or even a single site.

Reply:
:+1: